### PR TITLE
Dev/projects api

### PR DESCRIPTION
**Pull request by grong0**
it's done !!! check it out

### CODE REVIEW

File: next/prisma/schema.prisma [hunk 1]
```diff
@@ -2,280 +2,306 @@
...
 
-  //isPinned is a boolean value to determine if a user is pinned or not
-  isPinned Boolean @default(false)
+    //isPinned is a boolean value to determine if a user is pinned or not
+    isPinned Boolean @default(false)
 }
 
 //Account model represents an account in the system.
-//It contains information about the user, the type of account, and the authentication provider.
+//It contains information about the user, the type of account, and the authentication `vider.
```
Comment (PokeJofeJr4th) [inline next/prisma/schema.prisma:245]: why?

**Review by PokeJofeJr4th — CHANGES_REQUESTED** [review]
Have you manually tested? I'm worried the GET routes wouldn't have the right amount of data returned without explicitly using the select clause.

File: next/prisma/schema.prisma (hunk 1 above)
Comment (grong0) [reply next/prisma/schema.prisma:245]: i actually have no idea

Comment (grong0) [reply next/prisma/schema.prisma]: fixed i suppose

**Comment by grong0** [comment]
i did :)

**Comment by grong0** [comment]
Added implementation for contributers on projects.

**Comment by 2nPlusOne** [comment]
Noticed you're using a different indentation than the rest of the project. It should be 2 spaces indentation project-wide. Everyone should be set up with the same prettier formatting settings, but maybe that wasn't set up properly?

**Comment by grong0** [comment]
Yeah I noticed that too, lemme look into it.

**Comment by rtyocum** [comment]
There is no prettier config file in the repo at all. Without this prettier uses the user's default settings. If you want the consistency then someone needs to add the config file, otherwise you can expect the styles not to match as it's not documented anywhere (at least I couldn't find it)

**Comment by grong0** [comment]
Yeah, I just added one on my end.
@2nPlusOne is there a doc somewhere that mentions the specific code format? I didn't see a coding format mentioned on the Onboarding Guide or any Contribution section. Might've just missed it though.

**Comment by 2nPlusOne** [comment]
> Yeah, I just added one on my end. @2nPlusOne is there a doc somewhere that mentions the specific code format? I didn't see a coding format mentioned on the Onboarding Guide or any Contribution section. Might've just missed it though.

Not your fault! You're absolutely right, there isn't one. This is definitely something we should fix. [I created a ticket for this.](https://github.com/rit-sse/WebsiteTheSSEquel/issues/190) Feel free to edit it (either of you) with your additional thoughts.

**Comment by grong0** [comment]
Thanks!

**Pull request merged**